Common/Common.s

```diff
@@ -395,6 +395,21 @@ add \reg, r3, r4
 .set CFOptionsAddress, RtocAddress - ControllerFixOptions
 .set GeckoHeapPtr, 0x80005600
 
+# Internal scenes
+.set SCENE_VERSUS_CSS, 0x0200
+.set SCENE_VERSUS_SSS, 0x0201
```

Comment:
Am I crazy or are these both wrong?

Reply:
You're right, I think I entered them with the major and minor flipped. I'll update this.